pwa/src/components/common/elementCreationNew.tsx

```diff
@@ -77,32 +77,42 @@ const ElementCreationNew: React.FC<ElementCreationNewProps> = ({id, label, data,
 
       <div className="row">
         <h5>Create {label}</h5>
-        <div className="col-10">
           {
             select ? (
-              <div className="form-group">
-                <SelectInputComponent
-                  options={options !== null && options.length > 0 ? options : []}
-                  name={selectName} id={`${selectName}Input`}
-                  nameOverride={label}
-                  onChange={(e) => setValue(e.target.value)}
-                />
-              </div>
+              <>
+                <div className="col-10">
+                  <SelectInputComponent
+                    options={options !== null && options.length > 0 ? options : []}
+                    name={selectName} id={`${selectName}Input`}
+                    nameOverride={label}
```

Comment:
This label pushes the `select` down, and that's how the margin is created. That being said, in the other implementation of `ElementCreation` we opted to do it without a label. Which I would suggest here as well, because this would be the third label, which seems excessive.

![image](https://user-images.githubusercontent.com/66552932/153555016-effebd85-0bc7-4d1a-940c-0a3f6413b214.png)